include/bbp/sonata/report_reader.h

```diff
@@ -46,13 +47,25 @@ class SONATA_API SpikeReader
          */
         std::tuple<double, double> getTimes() const;
 
+        /**
+         * Create the spikes from the vectors of node_ids and timestamps
+         */
+        Spikes createSpikes() const;
```

Comment:
I think this should be a private function; since we don't want to have to support more API than necesary.